pyproject.toml

```diff
@@ -1,9 +1,9 @@
-[tool.black]
-target-version = ['py36']
+[tool.ruff]
+target-version = "py36"
```

Comment:
py36 isn't a valid option and looks like it never will be - https://github.com/astral-sh/ruff/issues/3826#issuecomment-1794880887

Reply:
good spot